server/util/status/status.go

```diff
@@ -10,131 +10,147 @@ import (
 func OK() error {
 	return status.Error(codes.OK, "")
 }
```

Comment:
would it make sense to have `IsOK`?

Reply:
We don't use it anywhere so I left it out. I think instead of using OK we return err == nil in most places.